p4-14/v1.0.5/tex/p4.pt

```diff
@@ -1830,10 +1835,6 @@ If the \texttt{header_instance} is not an element in a header stack, then the in
 header instance is marked invalid. It will not be available for matching in 
 subsequent \matchaction stages. The header will not be serialized on egress. 
 All field values in the header instance become uninitialized.
-
-If the \texttt{header_instance} is an element in a header stack, the effect is to 
-pop the indicated element from the stack. Any valid instances in the stack 
-at higher indices are copied to the next lower index.
```

Comment:
Similar comment here as for add_header: I thought the intended new behavior was to make the one element of the header stack specified as the argument to the remove_header() operation become invalid, if it was valid before.  Such an operation would have no effect on any other elements of the header stack.

Reply:
Fixed the same way as with `add_header`